sass/chosen.scss

```diff
@@ -70,6 +70,7 @@ $chosen-sprite-retina: url('chosen-sprite@2x.png') !default;
       cursor: pointer;
       opacity: 0;
       position: absolute;
+      width: 0;
```

Comment:
This small CSS change causes the carat to disappear in IE11. Why does the carat appear on-screen for a transparent input in the first place? I don’t have a good answer to that.